packages/core/src/server-runtime-client.ts

```diff
@@ -78,8 +78,7 @@ export class ServerRuntimeClient<
   /**
    * @inheritDoc
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public captureException(exception: any, hint?: EventHint, scope?: Scope): string {
+  public captureException(exception: unknown, hint?: EventHint, scope?: Scope): string {
```

Comment:
I think this might be breaking in a weird way but I can't think of an example right now.

Reply:
this should be fine as _accepting_ stuff as unknown like this will generally work for anything. The problematic part is replacing a return value of `any` with `unknown`, or with function args taking `...args: unknown[]` vs `...args: any[]`.